src/Holycode.Configuration/WebConfigExtensions.cs

```diff
@@ -43,19 +45,19 @@ public static IConfigurationBuilder AddWebConfig(this IConfigurationBuilder conf
 
         public static IConfiguration AddWebConfig(this IConfiguration configuration)
         {
-            var keys = ConfigurationManager.AppSettings.AllKeys;
+            var keys = ConfigManager.AppSettings.AllKeys;
```

Comment:
nazwa `ConfigurationManager` była ok, po co to zmieniać?

Reply:
Bo jest konflikt nazw typów: cała ta klasa jest zdefiniowana w namespace Microsoft.Extensions.Configuration, która zawiera już typ `ConfigurationManager` a my chcemy tu się dostać do `ConfigurationManager` z innego namespace